app/views/hyku_addons/account_settings/settings/_boolean_field_content.html.erb

```diff
@@ -2,7 +2,7 @@
 <%=f.fields_for :settings do |ff| %>
   <%= ff.label field_name %><br>
   <%= ff.label t("settings.helptexts.#{field_name}") %><br>
-  <%= ff.check_box field_name, {checked: @account.settings[field_name]}, true, false %>
+  <%= ff.check_box field_name, {}, "true", "false" %>
```

Comment:
I see you switched these values from booleans to Strings.  This appears to break the following test.  I'm curious why you made this change and whether they should get switched back to booleans or the test updated to reflect the type change.

>  1) HykuAddons::AccountBehavior Settings Customisations settings from environment variable check all boolean truthy values
     Failure/Error: expect(account.settings[key]).to be_a(TrueClass)
       expected "true" to be a kind of TrueClass
     # ./vendor/bundle/ruby/2.7.0/gems/rspec-support-3.10.1/lib/rspec/support.rb:97:in `block in <module:Support>'
     # ./vendor/bundle/ruby/2.7.0/gems/rspec-support-3.10.1/lib/rspec/support.rb:106:in `notify_failure'
     # ./vendor/bundle/ruby/2.7.0/gems/rspec-expectations-3.10.1/lib/rspec/expectations/fail_with.rb:35:in `fail_with'
     # ./vendor/bundle/ruby/2.7.0/gems/rspec-expectations-3.10.1/lib/rspec/expectations/handler.rb:40:in `handle_failure'
     # ./vendor/bundle/ruby/2.7.0/gems/rspec-expectations-3.10.1/lib/rspec/expectations/handler.rb:56:in `block in handle_matcher'
     # ./vendor/bundle/ruby/2.7.0/gems/rspec-expectations-3.10.1/lib/rspec/expectations/handler.rb:27:in `with_matcher'
     # ./vendor/bundle/ruby/2.7.0/gems/rspec-expectations-3.10.1/lib/rspec/expectations/handler.rb:48:in `handle_matcher'
     # ./vendor/bundle/ruby/2.7.0/gems/rspec-expectations-3.10.1/lib/rspec/expectations/expectation_target.rb:65:in `to'
     # ./spec/models/concerns/hyku_addons/account_behavior_spec.rb:131:in `block (5 levels) in <top (required)>'
     # ./spec/models/concerns/hyku_addons/account_behavior_spec.rb:129:in `each'
     # ./spec/models/concerns/hyku_addons/account_behavior_spec.rb:129:in `block (4 levels) in <top (required)>'

Reply:
I made this change because the first version wasn't saving as false, it might have been due to the line I removed above?

Reply:
Ill look into it tomorrow morning